format/midi/testdata/format-0.fqtest

```diff
@@ -0,0 +1,269 @@
+     |                                               |                |        [14]{}: time_signature 0xae-0xb6 (8)
+     |                                               |                |          time{}: 0xae-0xaf (1)
+0x0a0|                                          00   |              . |            delta: 0 0xae-0xaf (1)
+     |                                               |                |            tick: 0
+0x0a0|                                             ff|               .|          event: "time_signature" (65368) 0xaf-0xb1 (2)
+0x0b0|58                                             |X               |
+     |                                               |                |          signature{}: 0xb1-0xb6 (5)
+0x0b0|   04 04 02 18 08                              | .....          |            bytes: "[4 2 24 8]" 0xb1-0xb6 (5)
+     |                                               |                |            numerator: 4
+     |                                               |                |            denominator: 4
+     |                                               |                |            ticks_per_click: 24
+     |                                               |                |            thirty_seconds_per_quarter: 8
+     |                                               |                |        [15]{}: key_signature 0xb6-0xbc (6)
+     |                                               |                |          time{}: 0xb6-0xb7 (1)
+0x0b0|                  00                           |      .         |            delta: 0 0xb6-0xb7 (1)
+     |                                               |                |            tick: 0
+0x0b0|                     ff 59                     |       .Y       |          event: "key_signature" (65369) 0xb7-0xb9 (2)
+0x0b0|                           02 00 01            |         ...    |          key: "A minor" (1) 0xb9-0xbc (3)
+     |                                               |                |        [16]{}: sequencer_specific_event 0xbc-0xc6 (10)
+     |                                               |                |          time{}: 0xbc-0xbd (1)
+0x0b0|                                    00         |            .   |            delta: 0 0xbc-0xbd (1)
+     |                                               |                |            tick: 0
+0x0b0|                                       ff 7f   |             .. |          event: "sequencer_specific_event" (65407) 0xbd-0xbf (2)
+     |                                               |                |          info{}: 0xbf-0xc6 (7)
+0x0b0|                                             06|               .|            bytes: "[0 0 59 58 76 94]" 0xbf-0xc6 (7)
+0x0c0|00 00 3b 3a 4c 5e                              |..;:L^          |
+     |                                               |                |            manufacturer: "Mark Of The Unicorn (MOTU)" ("003B")
```

Comment:
To continue on the above comment: could this for example use a `d.FieldUintFn` and then map using the manufacturers mapper (but then maybe changed into a uint to str map)?

Reply:
Fixed in [578b7e78e395f4c1baefd8adb2cd6794176f2d2b](https://github.com/wader/fq/pull/1004/commits/578b7e78e395f4c1baefd8adb2cd6794176f2d2b)